main.go

```diff
@@ -63,6 +67,9 @@ func (p *Parser) ParseDir(targetDir string) *Result {
 					res.Function++
 					if x.Name.IsExported() {
 						res.ExportedFunction++
+						if strings.HasPrefix(x.Name.String(), "Test") {
+							res.Tests++
```

Comment:
I think this needs to check for the testing.t function argument or it could have false positives.